lasso/dimred/svd/html_str_eles.py

```diff
@@ -4,7 +4,7 @@

```

Comment:
Any idea what happened here?

Reply:
Ah I see it is the menu issue again. Did you verify it works fine?

Reply:
Ah, I edited my PR description after your review, you're too fast ! So:

> The best way to view the diff in `lasso/dimred/svd/html_str_eles.py` is `git diff --word-diff-regex=. origin/main -- lasso/dimred/svd/html_str_eles.py` (the only change is `menue` -> `menu`).